### PR TITLE
Move jvm11 options to jvm8 for DSE

### LINKED ISSUE #755: K8SSAND-1875 ⁃ JVM options for DSE should map to the jvm8 file, not the jvm11 one
For now we run DSE using jdk8 images, but have mappings for the options that put them in the jvm11 file.
We should pin these options to the jvm8 file and later figure out a way to dynamically point to the right file, depending on the jdk in use.

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1875) by [Unito](https://www.unito.io)
┆Issue Number: K8SSAND-1875
┆Priority: Medium
┆Link To Issue: https://k8ssandra.atlassian.net/browse/K8SSAND-1875

### PR DESCRIPTION
**Pull request by adejanovski**
**What this PR does**:
Redirects jvm11-server.options entries to the jvm8-server.options file for DSE.

**Which issue(s) this PR fixes**:
Fixes #755

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: config/crd/bases/k8ssandra.io_k8ssandraclusters.yaml [hunk 1]
```diff
@@ -8713,7 +8713,7 @@ spec:
                             containerImage:
                               default:
                                 repository: stargateio
-                                tag: v1.0.66
+                                tag: v1.0.67
```
Comment (adutra) [inline config/crd/bases/k8ssandra.io_k8ssandraclusters.yaml:8716]: Changes in this file look unrelated.

File: config/crd/bases/stargate.k8ssandra.io_stargates.yaml [hunk 2]
```diff
@@ -1104,7 +1104,7 @@ spec:
               containerImage:
                 default:
                   repository: stargateio
-                  tag: v1.0.66
+                  tag: v1.0.67
```
Comment (adutra) [inline config/crd/bases/stargate.k8ssandra.io_stargates.yaml:1107]: Same.

**Review by adutra — CHANGES_REQUESTED** [review]
Glad to see that the tag syntax is flexible enough to allow this. I only had concerns regarding unwanted changes in some files.

File: config/crd/bases/stargate.k8ssandra.io_stargates.yaml (hunk 2 above)
Comment (adejanovski) [reply config/crd/bases/stargate.k8ssandra.io_stargates.yaml:1107]: yeah, I think my previous PR which upgraded stargate hadn't rebuilt the CRDs manifests :( 
That's why it's here.
I'll ninja fix this

**Review by adutra — APPROVED** [review]
